src/org/labkey/cds/data/steps/CDSImportTask.java

```diff
@@ -99,16 +100,16 @@ public RecordedActionSet run(@NotNull PipelineJob job) throws PipelineJobExcepti
     {
         String dir = settings.get(DIRECTORY);
 
-        if (dir.contains(PIPELINE_TOKEN))
+        if (dir.contains(PATH_TOKEN))
         {
-            PipeRoot pipelineRoot = PipelineService.get().getPipelineRootSetting(containerUser.getContainer());
-            if (null != pipelineRoot)
+            String importFolderPath = CDSManager.get().getCDSImportFolderPath(containerUser.getContainer());
+            if (null != importFolderPath)
             {
-                dir = dir.replace(PIPELINE_TOKEN, pipelineRoot.getRootPath().getPath());
+                dir = dir.replace(PATH_TOKEN, importFolderPath);
             }
             else
             {
-                throw new PipelineJobException(PIPELINE_TOKEN + " was found in the path but the pipeline root has not been established for folder: " + containerUser.getContainer().getPath());
+                throw new PipelineJobException(PATH_TOKEN + " has not been established for folder: " + containerUser.getContainer().getPath());
```

Comment:
Not sure this error message is clear, I think you may want to use CDSModule.CDS_IMPORT_PATH here instead of PATH_TOKEN.